zerver/tests/test_event_system.py

```diff
@@ -1174,7 +1174,7 @@ def test_queries(self) -> None:
         realm = get_realm_with_settings(realm_id=user.realm_id)
 
         with (
-            self.assert_database_query_count(43),
+            self.assert_database_query_count(41),
```

Comment:
The decrease in the query count is 2 instead of 3 because `signup_announcements_stream` is set to None. So query count for that was not included in the first place.